qdrant_client/local/local_collection.py

```diff
@@ -140,6 +143,10 @@ def __init__(
         self.storage = None
         self.config = config
         if location is not None:
+            deprecation_warning_once(
+                f"Local mode is not recommended for collections with more than {self.LARGE_DATA_THRESHOLD:,} points. Consider using Qdrant server mode for better performance with large datasets.",
```

Comment:
It's probably unclear what is `server` mode, let's write about hosting qdrant in docker or using cloud